.travis.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       before_script:
         - doit env_create --name=pyviz --python=$PYENV_VERSION
         - source activate pyviz
-        - travis_wait 20 doit develop_install $CHANS_DEV -o recommended
+        - travis_wait 20 doit develop_install $CHANS_DEV -o tests -o examples -o graphs -o geo -o indirect
```

Comment:
This was previously `--all-extras`, but that was removed in #152 and replaced with just `-o recommended`. I can't quite follow what the intention is.

Reply:
recommended isn't including fastparquet, which is listed under `indirect` dependencies. The correct solution is to move fastparquet in another section if I was guessing.

Reply:
Short version:

Unless you're trying to change how things work in some way (which I'd be fine with, but you'd need to give me some background for me to help out with that), I *think* some of #152 should be reverted:
  1. Restore the pinning and use of 'all extras': https://github.com/pyviz/pyviz/pull/152/files#diff-354f30a63fb0907d4ad57269548329e3L49
  2. Restore installation of pyctdev from `-c pyviz/label/dev`

I.e. the important bit to keep is the minimum version pin of pyctdev to avoid a very old version of pyctdev being installed.


Long version!

I *think* #152  might have come from a misunderstanding, which comes from an unfortunate conjunction of complications:

  1. Originally pyctdev did not pin conda build. However, that didn't work out well because of variability in conda-build behavior over time. So pyctdev pinned a specific conda build. But...that version of conda build has no python 3.7 build. 
  2. When you install miniconda these days you get python 3.7. Until recently, conda install pyctdev in a fresh miniconda install would go for a recent pyctdev and downgrade python 3.7 to 3.6. That was conda <=4.5.11 I think. conda 4.5.12 changes that to instead installing an old pyctdev (presumably from before conda build was pinned by pyctdev) and sticking with python 3.7.

The above only affects the root/base environment (i.e. you can still make and test python 3.7 packages with pyctdev).

Reply:
And yes, pyctdev should be updated to use a newer conda-build. I am working on that. Unfortunately I am working on that at the same time as a new version of pyctdev - the purpose of which is to stop using some problematic/outdated features of conda build.